### PR TITLE
Http client fix encoding for osrm

### PR DESCRIPTION
**Pull request by milchakov**

### CODE REVIEW

**Comment by milchakov** [comment]
@bykoianko @syershov PTAL

File: platform/http_client.cpp [hunk 1]
```diff
@@ -10,7 +10,12 @@ namespace platform
 {
 HttpClient::HttpClient(string const & url) : m_urlRequested(url)
 {
+// Http client for Linux is not supported gzip encoding.
```
Comment (mpimenov) [inline platform/http_client.cpp]: does not support?

Comment (milchakov) [inline platform/http_client.cpp]: а, да

**Comment by bykoianko** [comment]
LGTM

**Pull request merged**